console/src/test/java/nl/nn/adapterframework/management/web/ApiExceptionTest.java

```diff
@@ -28,8 +28,8 @@ public class ApiExceptionTest {
 
 	public static List<?> data() {
 		return Arrays.asList(new Object[][] {
-			{"cannot configure", new ConfigurationException("cannot configure")},
-			{"cannot configure: (IllegalStateException) something is wrong", new ConfigurationException("cannot configure", new IllegalStateException("something is wrong"))},
+			{"cannot configure", new IbisException("cannot configure")},
+			{"cannot configure: (IllegalStateException) something is wrong", new IbisException("cannot configure", new IllegalStateException("something is wrong"))},
```

Comment:
NB: Er zijn verschillende tests met annotation `@ParameterizedTest` die geen parameters hebben. Zou je die tests gelijk kunnen fixen? Of goed omzetten, of de `@ParameterizedTest` annotation vervangen door gewoon `@Test` en `@MethodSource` weghalen.